interp/notation.ml

```diff
@@ -1180,8 +1190,9 @@ let interp_prim_token_gen ?loc g p local_scopes =
   try
     let (pat,loc), sc = find_interpretation p_as_ntn (find_prim_token ?loc g p) scopes in
     pat, (loc,sc)
-  with Not_found ->
```

Comment:
Noise.

Reply:
`find_interpretation` is complex enough as for us to want to keep the backtrace.